projects/sandbox/train/train.py

```diff
@@ -29,30 +47,29 @@ def load_background(background_path: Path, ifos):
 
 def load_signals(waveform_dataset: Path, parameter_names: List[str]):
     """
-    Load in signals and intrinsic parameters.
-    If no intrinsic parameters are requested, return None.
+    Load in validation signals and parameters
```

Comment:
Let's provide more context here for later that we are loading validation data so that we generate signals using lalsimulation/lalinference.